frontend/VolunteerOne/components/EventCard/CardHeader.js

```diff
@@ -0,0 +1,103 @@
+import { StyleSheet } from "react-native";
+import { Avatar } from "@rneui/themed";
```

Comment:
Let's sync on which package to use for the Avatar component, I'm currently using `@react-native-material/core`